ShovIP/app/src/main/java/com/example/shovip/MainActivity.kt

```diff
@@ -16,13 +17,12 @@ class MainActivity : AppCompatActivity() {
 
     private lateinit var appBarConfiguration: AppBarConfiguration
     private lateinit var binding: ActivityMainBinding
-
+    lateinit var sharedPreferences : SharedPreferences
```

Comment:
@shovian 

- It doesn't seem like the `SharedPreferences` are being used yet, do you plan to add this later?
- It might be a good idea to move the `SharedPreferences` to the `SettingsFragment`